### ISSUE TITLE
`make html` fails

### ISSUE THREAD

**Issue by djmitche** [mozilla/build-relengdocs#5]
```
Exception occurred:
  File "/home/dustin/code/moz/t/relengdocs/sandbox/lib/python2.7/site-packages/sphinx/writers/html.py", line 353, in depart_caption
    node.parent.traverse(nodes.image)[0], _('Permalink to this image'))
IndexError: list index out of range
The full traceback has been saved in /tmp/sphinx-err-6HqZwg.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
Makefile:53: recipe for target 'html' failed
make: *** [html] Error 1
```

This is with a fresh `pip install -r rtfd-requirements.txt`

**Comment by hwine** [comment]
docs build almost clean with sphinx-1.2.3 -- only warning is:
```
copying static files... WARNING: html_static_path entry u'/private/tmp/build-relengdocs/_static' does not exist
```

**Comment by hwine** [comment]
Looks like a variant of sphinx-doc/sphinx#1788

**Comment by hwine** [comment]
Closed with @64bf728
